lib/openid_connect_key_validation.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class OpenidConnectKeyValidation
+  # @param [private_key] OpenSSL::PKey
+  # @param [public_key] OpenSSL::PKey
+  def self.valid?(private_key:, public_key:, data:)
```

Comment:
What if we provided a default `data` generator? the bit in `app_artifacts.rb` seems arbitrary and isn't clear what it's used for
```suggestion
  def self.valid?(private_key:, public_key:, data: SecureRandom.hex)
```

Reply:
yep, makes sense